recipes/wxpython/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "wxPython" %}
+{% set version = "3.0.2.0" %}
+{% set sha256 = "d54129e5fbea4fb8091c87b2980760b72c22a386cb3b9dd2eebc928ef5e8df61" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+   fn: wxPython-src-{{ version }}.tar.bz2  # [not win]
+   url: http://sourceforge.net/projects/wxpython/files/wxPython/{{ version }}/wxPython-src-{{ version }}.tar.bz2  # [not win]
+   sha256: {{ sha256 }}  # [not win]
```

Comment:
I'd like to keep some of that there in case anyone wants to enable it in the future.

The os-x recipe used to work -- so it's close.